### PR TITLE
Add CDN url to readme

### LINKED ISSUE #971: Publish the library via cdn
https://cdnjs.com/libraries or https://www.cloudflare.com/features-cdn/#how-can-cloudflare-offer-a-free-service would be great.

### PR DESCRIPTION
**Pull request by zxqfox**
Fixes #971

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,6 +16,11 @@ as well as deobfuscate scripts processed by
 # Usage
 You can beautify javascript using JS Beautifier in your web browser, or on the command-line using node.js or python.
 
+To use in web browser include the script tag below in your document
+```html
+<script src="https://cdn.rawgit.com/beautify-web/js-beautify/master/js/lib/beautify-html.js"></script>
```
Comment (bitwiseman) [inline README.md]: This will not really work fully.  People will need the css and js beautifiers as well.

Comment (zxqfox) [inline README.md]: Something like this?
```html
<script src="https://cdn.rawgit.com/beautify-web/js-beautify/master/js/lib/beautify.js"></script>
<script src="https://cdn.rawgit.com/beautify-web/js-beautify/master/js/lib/beautify-css.js"></script>
<script src="https://cdn.rawgit.com/beautify-web/js-beautify/master/js/lib/beautify-html.js"></script>
```

File: README.md [hunk 2]
```diff
@@ -149,7 +156,7 @@ You might notice that the CLI options and defaults hash aren't 100% correlated.
 ## Directives to Ignore or Preserve sections (Javascript only)
 
 Beautifier for  supports directives in comments inside the file.
-This allows you to tell the beautifier to preserve the formtatting of or completely ignore part of a file.  
+This allows you to tell the beautifier to preserve the formtatting of or completely ignore part of a file.
```
Comment (zxqfox) [inline README.md]: Ouch.

Comment (bitwiseman) [inline README.md]: Thanks for cleaning that up. 😄

File: README.md [hunk 3]
```diff
@@ -16,6 +16,13 @@ as well as deobfuscate scripts processed by
 # Usage
 You can beautify javascript using JS Beautifier in your web browser, or on the command-line using node.js or python.
 
+To use in web browser include the script tag below in your document
+```html
+<script src="https://cdn.rawgit.com/beautify-web/js-beautify/master/js/lib/beautify.js"></script>
+<script src="https://cdn.rawgit.com/beautify-web/js-beautify/master/js/lib/beautify-css.js"></script>
+<script src="https://cdn.rawgit.com/beautify-web/js-beautify/master/js/lib/beautify-html.js"></script>
```
Comment (bitwiseman) [inline README.md]: It looks like you'll need to use a specific version and add something to the release.sh script to update this.

https://github.com/rgrove/rawgit/wiki/Frequently-Asked-Questions#how-long-does-the-cdn-cache-files-how-can-i-make-it-refresh-my-file 

Also, the cdn in not guaranteed to be up.  Please add a disclaimer of some kind or at least point to the FAQ so people understand the limits of this functionality.

**Comment by bitwiseman** [comment]
I think you have a great idea here.  But it might be worth taking the steps needed to use cdnjs.com as mentioned in your bug instead of rawgit.  Or do both.

**Comment by zxqfox** [comment]
> But it might be worth taking the steps needed to use cdnjs.com

I'd say if we want to use cdnjs.com we should publish to it automatically on publishing new version. It's MUCH harder that just add a link to readme. 😊

File: README.md (hunk 3 above)
Comment (zxqfox) [inline README.md]: Thanks for pointing on this. I though it refreshes once per day or something like that.

Replaced master with version and added replacer to release script.

**Comment by bitwiseman** [comment]
Okay, this change looks great! Could I ask you to open a new issue for publishing to cdnjs.com so that we can track progess on that?

**Pull request merged**